cf-agent/nfs.c

```diff
@@ -567,7 +615,19 @@ PromiseResult VerifyMount(EvalContext *ctx, char *name, const Attributes *a, con
     PromiseResult result = PROMISE_RESULT_NOOP;
     if (!DONTDO)
     {
-        snprintf(comm, CF_BUFSIZE, "%s -o %s %s:%s %s", CommandArg0(VMOUNTCOMM[VSYSTEMHARDCLASS]), opts, host, rmountpt, mountpt);
+
+        if (strcmp(a.mount.mount_type, "panfs") == 0)
```

Comment:
`a` has been changed to a pointer, so this needs to be updated to `a->mount`. (Maybe @nickanderson 's mistake while cherry-picking.